recipes/mender-gateway/debian-master/control

```diff
@@ -0,0 +1,15 @@
+Source: mender-gateway
+Maintainer: Mender Team <mender@northern.tech>
+Section: admin
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
+
+Package: mender-gateway
+Architecture: any
+Depends: libc6 (>= 2.12)
+Description: Mender Gateway
+ Mender Gateway exposes the Mender Device APIs to local devices
+ running the Mender client and forwards the requests to an upstream
+ Mender Server. It can optionally enhance the APIs with additional
+ edge-specific features.
```

Comment:
This package should recommend `mender-client`.

Reply:
Good description, by the way. Thanks.

Reply:
Why should mender-gateway recommend mender-client?

Reply:
For later iterations, namely for "system" updates, we want it to recommend it so that we can update the gateway and its devices as one group. We discussed it briefly during [the presentation](https://docs.google.com/presentation/d/1c-NxdrJfOi_TZiKJa8A_674ILYZltEMbMMFNqE8bikU/edit#slide=id.g10567ff8f8a_0_12).

You could argue that is not necessary for now, but I think it is nevertheless harmless to recommend, as using `mender-client` we will also provide a user friendly way of configuring `mender-gateway` from the first iteration.

Reply:
Then maybe suggesting is better; please be aware that by default recommended packages are automatically installed.

Reply:
"suggesting" is better then, I agree. I didn't remember that "recommends" are installed by default, thanks!